doc/source/cluster/kubernetes/examples/image-resize-example.md

```diff
@@ -0,0 +1,107 @@
+(kuberay-image-resize-example)=
+
+# Image Resize Example
+
+In this guide, we show you how to run a sample Ray workload reading images from Google Cloud Storage and resizing them on Kubernetes infrastructure.
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) (>= 1.23), [Helm](https://helm.sh/docs/intro/install/) (>= v3.4), [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), and [Docker](https://docs.docker.com/engine/install/)
+* [Google Cloud Application Credential](https://cloud.google.com/docs/authentication/provide-credentials-adc)

```

Comment:
Do we really need to set up the credential?

Reply:
This line is not needed because here uses anonymous client.

https://github.com/ray-project/kuberay/blob/cb2914d20ed3f0e995d67f2ada6c5dad79b33ec2/ray-operator/config/samples/ray-data-image-resize/ray_data_image_resize.py#L29